src/VuetifyResource.vue

```diff
@@ -67,7 +67,7 @@
                     :open-on-hover="true"
                     :right="true"
                     :top="true"
-                    direction="bottom"
+                    :direction="speedDailDirection"
```

Comment:
It's `speed-dial` not `speed-dail`

Reply:
seems that `showSpeedDail` isn't spelled properly also haha